### ISSUE TITLE
Error while loading EvolvingGraphs.jl

### ISSUE THREAD

**Issue by juanpide** [EtymoIO/EvolvingGraphs.jl#10]
Hello.

Today I've installed the package
https://github.com/weijianzhang/EvolvingGraphs.jl

The installation didn't generate any error message.

But When I try to load it I get this error:

`using EvolvingGraphs`

```
WARNING: could not import Base.slice into EvolvingGraphs

WARNING: deprecated syntax "abstract AbstractGraph{V,T,E}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:9.
Use "abstract type AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:10.
Use "abstract type AbstractEvolvingGraph{V,T,E}<:AbstractGraph{V,T,E} end" instead.

WARNING: deprecated syntax "abstract AbstractStaticGraph{V,E}<:AbstractGraph{V,E}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:11.
Use "abstract type AbstractStaticGraph{V,E}<:AbstractGraph{V,E} end" instead.

WARNING: deprecated syntax "abstract AbstractPath" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:19.
Use "abstract type AbstractPath end" instead.

WARNING: deprecated syntax "typealias NodeType{V} Union{Node{V},AttributeNode{V},TimeNode{V}}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:84.
Use "NodeType{V} = Union{Node{V},AttributeNode{V},TimeNode{V}}" instead.

WARNING: deprecated syntax "typealias NodeVector{V} Vector{Node{V}}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:87.
Use "NodeVector{V} = Vector{Node{V}}" instead.

WARNING: deprecated syntax "typealias EdgeType{V} Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" at C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl:137.
Use "EdgeType{V} = Union{Edge{V},TimeEdge{V},WeightedTimeEdge{V}}" instead.
LoadError: LoadError: invalid subtyping in definition of AbstractStaticGraph
while loading C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\base.jl, in expression starting on line 10
while loading C:\Users\JOE\.julia\v0.6\EvolvingGraphs\src\EvolvingGraphs.jl, in expression starting on line 45

Stacktrace:
 [1] include_from_node1(::String) at .\loading.jl:569
 [2] include(::String) at .\sysimg.jl:14
 [3] include_from_node1(::String) at .\loading.jl:569
 [4] eval(::Module, ::Any) at .\boot.jl:235
 [5] _require(::Symbol) at .\loading.jl:483
 [6] require(::Symbol) at .\loading.jl:398
```

I don't know if it's a problem with my installation or with this version.

Regards

**Comment by jpfairbanks** [comment]
This is a series of warnings about syntax from Julia 0.5 that was deprecated for 0.6. The instructions that are part of the warnings are valid and should be corrected with a new version of the package being tagged for release on Metadata. I am sure that @weijianzhang would be happy to merge a PR bringing this package up to date for 0.6

**Comment by weijianzhg** [comment]
Thanks for letting me know! Just a bit busy with my thesis recently. Will fix it ASAP.

**Comment by jpfairbanks** [comment]
> Thanks for letting me know! Just a bit busy with my thesis recently. 

If we prioritized theses above Julia packages, then there would be no good Julia packages. ;)

**Comment by alexmorley** [comment]
Should be fixed in #12
